### PR TITLE
Bug Fix

### PR DESCRIPTION
**Pull request by kenpaxtonlim**
Fix incorrect socialmedia error message. #124 
Fix edit to allow socialmedia to be edited to an empty string. #136

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -117,12 +117,18 @@ public static int parsePositiveInteger(String num) throws IllegalValueException
      * Parses three {@code String facebook, twitter, instagram} into an {@code SocialMedia}.
      */
     public static SocialMedia parseSocialMedia(Optional<String> facebook,
-            Optional<String> twitter, Optional<String> instagram) throws IllegalValueException {
+            Optional<String> twitter, Optional<String> instagram, boolean isAdding) throws IllegalValueException {
         requireAllNonNull(facebook, twitter, instagram);
```
Comment (500poundbear) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: Hi @kenpaxtonlim , when will IllegalValueException be thrown?

**Review by 500poundbear — APPROVED** [review]
nice fix

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (kenpaxtonlim) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: Oh I forget to delete that, previously the constructor will throw it but that was removed.

**Comment by 500poundbear** [comment]
That's great!

**Pull request merged**